util/disk/disk_unix.go

```diff
@@ -0,0 +1,18 @@
+//go:build !windows && !openbsd
+// +build !windows,!openbsd
+
+package disk
+
+import (
+	"syscall"
+)
+
+func GetDiskSize(root string) (free int64, total int64, err error) {
+	var st syscall.Statfs_t
+	if err := syscall.Statfs(root, &st); err != nil {
+		return 0, 0, err
+	}
+	diskSize := int64(st.Bsize) * int64(st.Blocks)
+	freeSize := int64(st.Bsize) * int64(st.Bfree)
```

Comment:
Not super familiar with this API - do we want to use `free` or `available` here?